src/Sylius/Component/Attribute/Model/AttributeValue.php

```diff
@@ -144,13 +158,101 @@ public function getType()
     }
 
     /**
+     * @return boolean
+     */
+    public function getBoolean()
+    {
+        return $this->boolean;
+    }
+
+    /**
+     * @param boolean $boolean
+     */
+    public function setBoolean($boolean)
+    {
+        $this->boolean = $boolean;
+    }
+
+    /**
+     * @return string
+     */
+    public function getText()
+    {
+        return $this->text;
+    }
+
+    /**
+     * @param string $text
+     */
+    public function setText($text)
+    {
+        $this->text = $text;
+    }
+
+    /**
+     * @return int
+     */
+    public function getInteger()
+    {
+        return $this->integer;
+    }
+
+    /**
+     * @param int $integer
+     */
+    public function setInteger($integer)
+    {
+        $this->integer = $integer;
+    }
+
+    /**
+     * @return float
+     */
+    public function getFloat()
+    {
+        return $this->float;
+    }
+
+    /**
+     * @param float $float
+     */
+    public function setFloat($float)
+    {
+        $this->float = $float;
+    }
+
+    /**
+     * @return \DateTime
+     */
+    public function getDateTime()
+    {
+        return $this->dateTime;
+    }
+
+    /**
+     * @param \DateTime $dateTime
+     */
+    public function setDateTime($dateTime)
+    {
+        $dateTimeValue = sprintf('%d/%d/%d %d:%d',
```

Comment:
This is kind of workaround - we should make Symfony form type to return always \DateTime instance. Hm?